scene/gui/tree.cpp

```diff
@@ -3188,6 +3195,32 @@ bool Tree::_scroll(bool p_horizontal, float p_pages) {
 	return scroll->get_value() != prev_value;
 }
 
+Rect2 Tree::_get_scrollbar_layout_rect() const {
+	const Size2 size = get_size();
+	const Ref<StyleBox> background = theme_cache.panel_style;
+
+	const Rect2 content_rect = Rect2(background->get_offset(), size - background->get_minimum_size());
```

Comment:
This one is weird. I think we want to subtract the right and bottom margin instead of the min size. It probably worked by accident before, but StyleBoxes can override the min size now, and it would no longer represent the right-bottom offset. (Same in the original method where this was copied from and where this is still used too).

Reply:
I changed my comment here, took me a moment to unwrap what's going on.

Reply:
The parameters here are a position and a size. `background->get_minimum_size()` returns the sum of all margins introduced by the stylebox. So I don't think it assumes the equality margins.

Reply:
The code here _assumes_ that `background->get_minimum_size()` returns the sum of all margins, but that's not necessarily true. It's the default value/behavior, but it can be changed. So I'm asking you to change this, and other similar, part to subtract the margins explicitly rather than assume that the minimum size always equals to them.

Reply:
Makes sense. I'll update this PR tomorrow.